boards/hamilton/include/board.h

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2016 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_hamilton
+ * @ingroup     boards
+ * @brief       Support for the Hamilton board.
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Hamilton board
+ *
+ * @author      Michael Andersen <m.andersen@cs.berkeley.edu>
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    xtimer configuration
+ * @{
+ */
+#define XTIMER_DEV          TIMER_DEV(1)
+#define XTIMER_CHAN         (0)
+#define XTIMER_OVERHEAD     (0)
+/** @} */
+
+/**
+ * @name AT86RF233 configuration
+ * @{
+ */
+#define AT86RF2XX_PARAM_SPI      SPI_DEV(0)
+#define AT86RF2XX_PARAM_SPI_CLK  SPI_CLK_5MHZ
+#define AT86RF2XX_PARAM_CS       GPIO_PIN(PB, 31)
+#define AT86RF2XX_PARAM_INT      GPIO_PIN(PB, 0)
+#define AT86RF2XX_PARAM_SLEEP    GPIO_PIN(PA, 20)
+#define AT86RF2XX_PARAM_RESET    GPIO_PIN(PB, 15)
+/** @} */
+
+/**
+ * @name    LED pin definitions and handlers
+ * @{
+ */
+#define LED0_PIN                  GPIO_PIN(0, 19)
+
+#define LED_PORT                  PORT->Group[0]
```

Comment:
Should be LED0_PORT (you forgot this one)

Reply:
done